src/hooks/useEmails.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  useQuery,
+  type QueryFunction,
+  type QueryKey,
+  useMutation,
+} from '@tanstack/react-query';
+import axios, { type AxiosError } from 'axios';
+import { EMAILAPI_URL } from '../constants';
+import { type Email, type EmailData, type SendEmailsResponse } from '../types';
+
+const fetchEmails: QueryFunction<Email[], QueryKey> = async () => {
+  const response = await axios.get(`${EMAILAPI_URL}/api/v1/sent`);
+  return response.data.emails;
+};
+
+const sendEmail = async (emailData: EmailData): Promise<SendEmailsResponse> => {
+  const response = await axios.post(`${EMAILAPI_URL}/api/v1/send`, emailData);
+  return response.data;
+};
+
+export const useEmails = () => {
+  const useSendEmail = () => {
```

Comment:
Use prefix is used for hooks this is not a hook. 
```suggestion
  const sendEmail = () => {
```